support-e2e/tests/oneOffContributions.test.ts

```diff
@@ -13,6 +13,13 @@ interface TestDetails {
 
 const testsDetails: TestDetails[] = [
 	{ paymentType: 'Credit/Debit card', customAmount: '22.55' },
+	/**
+	 * PayPal is currently throwing a "to many login attempts" error, so we're
+	 * going to inactivate this test until we have a solution for it to avoid
+	 * alert numbness.
+	 *
+	 * TODO - re-enable this test when PayPal is fixed
+	 */
 	{ paymentType: 'PayPal' },
```

Comment:
Just need to comment out `{ paymentType: 'PayPal' }` too I presume?

Reply:
```suggestion
	// { paymentType: 'PayPal' },
```

Reply:
@georgeblahblah nice catch, thanks!